src/test/regress/input/multi_copy.source

```diff
@@ -107,6 +107,15 @@ SELECT count(*) FROM customer_copy_hash;
 -- Confirm that data was copied
 SELECT count(*) FROM customer_copy_hash;
 
+-- Make sure that master_update_shard_statistics() only updates shard length for
```

Comment:
for the sake of completeness. Can we also explicitly test for an append table ?
just set their min/max/shardsize values to arbitrary values (-1 ?). Then run the update utility to set the correct (assuming it is correct) values.